go/enclave/enclave.go

```diff
@@ -89,6 +92,15 @@ func NewEnclave(
 
 	nodeShortID := common.ShortAddress(config.HostID)
 
+	var prof *profiler.Profiler
+	if config.ProfilerEnabled {
```

Comment:
this should not be possible in secure mode.

Reply:
Good point - wonder if we should leave it to some sort of debug build ?

Maybe something to reiterate in the future? 

Removing the profiler from the enclave 👍

Reply:
Update: Will tie this to the secure execution of the enclave.